index.html

```diff
@@ -44,6 +44,7 @@
 <li><a href="{{ site.url }}users/grechanp/">Pavel Grechanuk</a> (<a href="https://github.com/grechasneak" class="user-mention">@grechasneak</a>)</li>
 <li><a href="{{ site.url }}users/schicklr/">Robert Schickler</a> (<a href="https://github.com/albeanth" class="user-mention">@robertschickler</a>)</li>
```

Comment:
@robertschickler Unrelated note, you may want to change this re-direction from Tony's page to yours...

Reply:
`href="https://github.com/albeanth" class="user-mention">@robertschickler</a>)</li>' is the line

Reply:
@ryanstwrt  looks good man. 

@harterj  I think it's good to go. The issue with Rob's redirect link isn't necessarily related to Ryan's page. I vote we push it through and just make sure Rob knows about his incorrect link and fix it later.